lib/index.js

```diff
@@ -30,37 +35,49 @@ const defaults = {
  */
 
 function StickyInlines(opts) {
-  opts = Object.assign({}, defaults, opts)
-  const { allowedTypes, bannedTypes, hasStickyBoundaries, canBeEmpty } = opts
+  opts = Object.assign({}, defaults, opts);
+  const { allowedTypes, bannedTypes, hasStickyBoundaries, canBeEmpty } = opts;
 
-  if (allowedTypes && !Array.isArray(allowedTypes)) { console.warn('slate-sticky-inline: allowedTypes must be an Array of Strings') }
-  if (!Array.isArray(bannedTypes)) { console.warn('slate-sticky-inlines: bannedTypes must be an Array of Strings') }
+  if (allowedTypes && !Array.isArray(allowedTypes)) {
+    console.warn(
+      'slate-sticky-inline: allowedTypes must be an Array of Strings'
+    );
+  }
+  if (!Array.isArray(bannedTypes)) {
+    console.warn(
+      'slate-sticky-inlines: bannedTypes must be an Array of Strings'
+    );
+  }
 
   /**
    * Keydown entry point.
    *
    * @param {Event} event
    * @param {Change} change
-   * @param {Editor} editor
+   * @param {Next} next
    * @return {Change}
    */
 
-  function onKeyDown(event, change, editor) {
+  function onKeyDown(event, change, next) {
     // We are working inside a specific inline, and they specifically said they don't want it to be sticky.
-    if (change.value.focusInline && isInlineBanned(change.value.focusInline, opts)) return null
+    if (
+      change.value.focusInline &&
+      isInlineBanned(change.value.focusInline, opts)
+    )
```

Comment:
please add { } around the return next(); expression if its going to be broken across multiple lines this way